### PR TITLE
RELATED: CL-9730 Fetching related data loads forever when combine both show in % and measure filter

### PR DESCRIPTION
**Pull request by dprentis**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded.

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul/172/ : SUCCESS in 1m 51s

File: src/catalogue.js [hunk 1]
```diff
@@ -44,9 +48,9 @@ function bucketItemsToExecConfig(bucketItems) {
         const maql = get(definition, 'metricDefinition.expression');
 
         if (maql) {
-            return maql.replace(/{[^}]+}/g, (match) => {
+            return maql.replace(ID_REGEXP, match => {
                 const expression = idToExpr[trim(match, '{}')];
-                return expression.substr(SELECT_LENGTH);
+                return expression.substr(SELECT_LENGTH).replace(WHERE_REGEXP, '');
```
Comment (vaclavbohac) [inline src/catalogue.js]: Please add test case for this

Comment (dprentis) [inline src/catalogue.js]: it is done

**Comment by gdgate** [comment]
Build succeeded.

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul/174/ : SUCCESS in 1m 54s

**Comment by gdgate** [comment]
Build succeeded.

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul/175/ : SUCCESS in 1m 47s

**Pull request merged**